src/libraries/icubmod/embObjMotionControl/embObjMotionControl.cpp

```diff
@@ -953,6 +955,16 @@ bool embObjMotionControl::fromConfig_Step2(yarp::os::Searchable &config)
         return false;
     }
 
+    ////// LUGRE PARAMETERS
+    if(! _mcparser->parseLugreGroup(config,_lugre_params))
+    {
+        yWarning() << "embObjMC " << getBoardInfo() << "LUGRE section: missing, sensorless torque estimation disabled";
```

Comment:
here the function returns false only if a bad format of `LUGRE` group is found. The case of missing `LUGRE` group is already done in the function itself.